include/SDL3/SDL_mouse.h

```diff
@@ -227,6 +259,21 @@ extern SDL_DECLSPEC const char * SDLCALL SDL_GetMouseNameForID(SDL_MouseID insta
  */
 extern SDL_DECLSPEC SDL_Window * SDLCALL SDL_GetMouseFocus(void);
 
+/**
+ * Set a user-defined function by which to transform relative mouse inputs,
+ * and its associated internal data which will be passed to the function.
```

Comment:
```suggestion
```
This was confusing, assuming there was internal state that needs to be modified. The callback pattern is pretty common in SDL and I think it's understood that the userdata pointer contains any state that you want associated with the callback.